packages/security/src/appSecurity/AppSecurity.ts

```diff
@@ -33,6 +32,20 @@ export class AppSecurity {
   }
 
   // TODO call init endpoint on go mobile security service AEROGEAR-8774
+  public clientInit(): Promise<any> {
+    const clientInitResponse = this.getClientData()
+    .then(metricsPayload => {
+      const initPayload: {[key: string]: any; } = {};
+      // build the initPayload
+      initPayload.deviceId = metricsPayload.clientID;
+      initPayload.appId = metricsPayload.app.appId,
+      initPayload.deviceType = metricsPayload.device.platform;
+      initPayload.deviceVersion = metricsPayload.device.platformVersion;
+      initPayload.version = metricsPayload.app.appVersion;
+      return axios.post(this.internalConfig.url, initPayload);
```

Comment:
[Question] This is probably service we will require better flexibility that metrics publisher do not offer for us?

[Question] Does URL contain the full path to service?

Reply:
[Answer] I think so
[Answer] yes see my version of showcase config file at https://github.com/austincunningham/ionic-showcase/blob/383dc01008d62b2e8af92a0888dc481c381b1788/src/mobile-services.js#L10

Reply:
@austincunningham Yes. I got into impression that there will be more than one endpoint but I think your case is simple as there will be the only endpoint called so the entire path can be used etc. Great work overall.

Reply:
@wtrocki I taught there was more that one as well, but I checked the acceptance criteria and just needed this one.